packages/core/bootstrap/test/unit/cache.test.ts

```diff
@@ -105,36 +110,175 @@ describe('cache', () => {
       await callAndExpect(counter, 3, 0)
     })
 
-    it(`invalidates cache - after configured minimum maxAge of 35s`, async () => {
-      context.cache.minimumAge = 1000 * 35
+    // it(`invalidates cache - after configured minimum maxAge of 35s`, async () => {
+    //   context.cache.minimumAge = 1000 * 35
 
-      const counter = await withCache()(counterFrom(0), context)
-      await callAndExpect(counter, 3, 0)
+    //   const counter = await withCache()(counterFrom(0), context)
+    //   await callAndExpect(counter, 3, 0)
+
+    //   clock.tick(1000 * 30 + 1)
+    //   await callAndExpect(counter, 1, 0)
+
+    //   clock.tick(1000 * 5 + 1) // extra 1ms
+    //   await callAndExpect(counter, 1, 1)
+    //   await callAndExpect(counter, 1000, 1)
+
+    //   clock.tick(1000 * 35 + 1) // extra 1ms
+    //   await callAndExpect(counter, 1, 2)
+    // })
+
+    // it(`will not set a TTL lower than default minimum TTL of 30s`, async () => {
+    //   const counter = await withCache()(counterFrom(0, { maxAge: 1000 * 10 }), context)
+    //   await callAndExpect(counter, 3, 0)
 
-      clock.tick(1000 * 30 + 1)
-      await callAndExpect(counter, 1, 0)
+    //   clock.tick(1000 * 5)
+    //   await callAndExpect(counter, 1, 0)
 
-      clock.tick(1000 * 5 + 1) // extra 1ms
-      await callAndExpect(counter, 1, 1)
-      await callAndExpect(counter, 1000, 1)
+    //   clock.tick(1000 * 5 + 1) // extra 1ms
+    //   await callAndExpect(counter, 1, 0)
+    //   await callAndExpect(counter, 1000, 0)
 
-      clock.tick(1000 * 35 + 1) // extra 1ms
-      await callAndExpect(counter, 1, 2)
+    //   clock.tick(1000 * 30 + 1) // extra 1ms
+    //   await callAndExpect(counter, 1, 1)
+    // })
```

Comment:
Why is this commented out?

Reply:
Shouldn't have been. Un-commented it out.